Dockerfile

```diff
@@ -1,13 +1,14 @@
-FROM remotepixel/amazonlinux-gdal:2.4.2
+FROM remotepixel/amazonlinux:gdal3.0-py3.7-build
```

Comment:
we now use the new amazonlinux docker container https://github.com/RemotePixel/amazonlinux